src/mainComponents/FilterTasks.jsx

```diff
@@ -0,0 +1,95 @@
+import Select from 'react-select';
+
+export default function FilterTasks({tasks, setCurrentFilter, currentFilter}) {
```

Comment:
"Filter" je glagol (je glupo je) - tako da bi bolje ime bilo nešto kao `TasksFilterSelect` ili nešto slično šta podrazumjeva da se neka komponenta/element vraća, a ne akcija filtriranja, ako me kužiš.